src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -170,6 +190,14 @@ class BlockchainImpl(
       mpt.get(address)
     }
 
+  override def getAccountStorageAt(rootHash: ByteString, position: BigInt): ByteString = {
+    val storageMpt =
+      MerklePatriciaTrie[UInt256, UInt256](rootHash.toArray[Byte], nodeStorage,
```

Comment:
This is done here as well https://github.com/input-output-hk/etc-client/blob/phase/4/jsonrpc/src/main/scala/io/iohk/ethereum/ledger/InMemoryWorldStateProxy.scala#L118